### PR TITLE
Update zh_cn.lang

### PR DESCRIPTION
**Pull request by Inerecraft**
update zh_cn.lang

### CODE REVIEW

File: src/main/resources/assets/threng/lang/zh_cn.lang [hunk 1]
```diff
@@ -1,53 +1,67 @@
...
+item.threng:material10.name=推测核心(8x)
+item.threng:material11.name=推测核心(16x)
+item.threng:material12.name=推测核心(32x)
+item.threng:material13.name=推测核心(64x)
 item.threng:material14.name=推测处理器
+
 tile.threng:machine0.name=福鲁伊克斯聚合器
 tile.threng:machine1.name=脉冲离心机
 tile.threng:machine2.name=ME电路板蚀刻器
 tile.threng:machine3.name=抢占式装配单元
```
Comment (magcicada) [inline src/main/resources/assets/threng/lang/zh_cn.lang:22]: Change to `优先装配单元` is better than before.

File: src/main/resources/assets/threng/lang/zh_cn.lang [hunk 2]
```diff
@@ -1,53 +1,67 @@
...
+tile.threng:machine4.name=ME级别维护器
```
Comment (magcicada) [inline src/main/resources/assets/threng/lang/zh_cn.lang:23]: Change to `ME缓存指令器` is better than before.

File: src/main/resources/assets/threng/lang/zh_cn.lang [hunk 3]
```diff
@@ -1,53 +1,67 @@
...
+
 threng.container.aggregator=福鲁伊克斯聚合器
 threng.container.centrifuge=脉冲离心机
 threng.container.etcher=ME电路板蚀刻器
 threng.container.fast_crafter=抢占式装配单元
+threng.container.level_maintainer=ME级别维护器
+threng.container.big_assembler=大型一体分子装配室
+threng.container.energizer=晶体充能器
+
+threng.misc.tooltip.request_size=维护量
```
Comment (magcicada) [inline src/main/resources/assets/threng/lang/zh_cn.lang:40]: And then here is `缓存量`.

File: src/main/resources/assets/threng/lang/zh_cn.lang [hunk 4]
```diff
@@ -1,53 +1,67 @@
...
+threng.misc.tooltip.batch_size=批量制作大小
```
Comment (magcicada) [inline src/main/resources/assets/threng/lang/zh_cn.lang:41]: At here,the SIZE means `数量`,`批量制作大小` should change to `批量制作数量`.

File: src/main/resources/assets/threng/lang/zh_cn.lang [hunk 5]
```diff
@@ -1,53 +1,67 @@
...
+tile.threng:big_assembler1.name=大型分子装配室外壁
+tile.threng:big_assembler2.name=大型分子装配室控制器
+tile.threng:big_assembler3.name=大型分子装配室样板核心
+tile.threng:big_assembler4.name=大型分子装配室处理核心
+tile.threng:big_assembler5.name=大型分子装配室IO端口
+
 threng.container.aggregator=福鲁伊克斯聚合器
 threng.container.centrifuge=脉冲离心机
 threng.container.etcher=ME电路板蚀刻器
 threng.container.fast_crafter=抢占式装配单元
```
Comment (magcicada) [inline src/main/resources/assets/threng/lang/zh_cn.lang:35]: The same as line 22.

File: src/main/resources/assets/threng/lang/zh_cn.lang [hunk 6]
```diff
@@ -1,53 +1,67 @@
...
+threng.container.level_maintainer=ME级别维护器
```
Comment (magcicada) [inline src/main/resources/assets/threng/lang/zh_cn.lang:36]: The same as line 23.

**Comment by phantamanta44** [comment]
i don't speak chinese, so i can't really make any judgement on the correctness of these reviews. assuming they're correct, could you open a new PR with the fixed changes, since the PR author seems to be inactive? @magcicada

**Pull request merged**